docs/aop/user/intro-to-aop/aop-overview.md

```diff
@@ -0,0 +1,51 @@
+---
+title: App Orchestration Platform overview
+description: Learn about the App Orchestration Platform and how you can use it.
+template: concept-topic-template
+---
+
+The App Orchestration Platform (AOP) lets [Spryker Cloud Commerce OS (SCCOS)](/docs/cloud/dev/spryker-cloud-commerce-os/getting-started-with-the-spryker-cloud-commerce-os.html) customers to connect, configure, and use the available third-party services, or apps, in their application with a click of a button, without development efforts from their side.
```

Comment:
```suggestion
The App Orchestration Platform (AOP) allows [Spryker Cloud Commerce OS (SCCOS)](/docs/cloud/dev/spryker-cloud-commerce-os/getting-started-with-the-spryker-cloud-commerce-os.html) customers to connect, configure, and use the available third-party services, or apps, in their application with a click of a button, without development efforts from their side.
```

Reply:
The Google developers style guide still recommends to use "let" instead of "allow" https://developers.google.com/style/word-list#allows-you-to